src/less/progress-bar/base/progress-bar.less

```diff
@@ -0,0 +1,25 @@
+progress[value] {
```

Comment:
We should not modify the base `progress`, but rather we should have a class on top of that to make it look like we want.
Like in checkbox, we have a `checkbox` class on top of the `input type="checkbox"`

Reply:
okay I tried that but I couldn't get it to work. I'll take another shot at it. The `progress` tag was surprisingly difficult to style..

Reply:
okay that worked. I don't know why it didn't seem to work earlier. Maybe because I wasn't targeting things correctly.